### PR TITLE
[TA] fix missing parameters

### PR DESCRIPTION
**Pull request by mssfang**
(1) Use `Integer` instead of `int` because `maxSentenceCount` is optional parameter. It is not a breaking change since we haven't release the feature.

(2) Added missing parameters assignment in the extractive summarization task.

(3) changed the test input of extractive summarization. The input sample is from service team's Extractive summarization Microsoft Doc.

(4) Added more sample for testing "top k sentences sorted by rankScore/offset"

API diff: https://apiview.dev/Assemblies/Review/f62ee261035a44189c597868c7a8a7ee?diffRevisionId=a79e108085404d939026dbd3653a5988&doc=False&diffOnly=True

### CODE REVIEW

**Review by kristapratico — APPROVED** [review]
LGTM

File: sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TestUtils.java [hunk 1]
```diff
@@ -527,28 +525,34 @@ static ExtractKeyPhrasesResultCollection getExpectedBatchKeyPhrases() {
...
+                1.0, 0, 160),
+            getExpectedSummarySentence(
+                "In my role, I enjoy a unique perspective in viewing the relationship among three attributes of human"
+                    + " cognition: monolingual text (X), audio or visual sensory signals, (Y) and multilingual (Z).",
+                0.958, 324, 192),
+            getExpectedSummarySentence(
+                "At the intersection of all three, there’s magic—what we call XYZ-code as illustrated in Figure"
+                    + " 1—a joint representation to create more powerful AI that can speak, hear, see, and understand"
+                    + " humans better.",
+                0.929, 517, 203)
```
[the remark this answers is not in the document]
Comment (mssfang) [reply sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TestUtils.java:552]: That is right. The rank score here is for view but will not be used for verification.

File: sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeActionsAsyncClient.java [hunk 2]
```diff
@@ -279,7 +280,15 @@ private JobManifestTasks getJobManifestTasks(TextAnalyticsActions actions) {
                         final ExtractiveSummarizationTask extractiveSummarizationTask =
                             new ExtractiveSummarizationTask();
                         extractiveSummarizationTask.setParameters(
-                            new ExtractiveSummarizationTaskParameters().setModelVersion(action.getModelVersion()));
+                            new ExtractiveSummarizationTaskParameters()
```
Comment (maririos) [inline sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeActionsAsyncClient.java:283]: yay!!! do you have an issue for adding the properties to the other actions?

**Review by maririos — APPROVED** [review]

File: sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeActionsAsyncClient.java (hunk 2 above)
Comment (mssfang) [reply sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeActionsAsyncClient.java:283]: Yes. I think the first might go to the GA version first, then merge it to latest beta.

Comment (maririos) [reply sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeActionsAsyncClient.java:283]: ahh good point

**Pull request merged**